python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_logistic.py

```diff
@@ -116,22 +224,18 @@ def logisticModel(X):
 # -------------------------------------
 
 # %%
-timeObservationsVector = ot.Sample([[timeObservations[i, 0] for i in range(nbobs)]])
-timeObservationsVector[0:10]
-
-# %%
-populationObservationsVector = ot.Sample(
-    [[populationObservations[i, 0] for i in range(nbobs)]]
-)
-populationObservationsVector[0:10]
-
-# %%
-# The reference values of the parameters.
+# The calibration algorithm takes input and output samples as arguments.
+# In this case, we choose to consider one single observation in dimension 22.
+# In order to perform calibration, we create a `Sample` of input times which has
+# one observation in dimension 22.
+timeObservationsSample = ot.Sample([timeObservations.asPoint()])
+timeObservationsSample[0, 0:5]
 
 # %%
-a = 0.03134
-c = -22.58
-thetaPrior = [a, c]
+# Similarly, we create a `Sample` of output populations which has
```

Comment:
In this area, there are weird empty spaces. Can you check what is going on? I suspect it has something to do with missing empty lines between RST and Python code. Another possibility is the lack of prints, which makes the output look like RST text to Sphinx - see my next comment.

Reply:
Ok. Fixed.